tests/functional/testHeartbeat.js

```diff
@@ -27,16 +27,18 @@ describe('Backend failure', function() {
     await vpn.wait();
   });
 
-  it('Backend failure in the help menu', async () => {
-    await vpn.waitForElementProperty('getHelpLink', 'visible', 'true');
-    await vpn.clickOnElement('getHelpLink');
-    await vpn.waitForElementProperty('getHelpLink', 'visible', 'false');
+  /* TODO
```

Comment:
What's the reason for this test being commented out? We should either document the reason or delete the code.

Reply:
I'm about to file an issue. This test is totally broken but fixing it is out of scope.

Reply:
When this PR is reviewed, before landing I'll file the issue and I'll write the ID here.